### PR TITLE
PORT-8075: Added api key approval template

### PR DESCRIPTION
**Pull request by Meet-Fiserv**

### CODE REVIEW

File: email-templates/api-key-approval.template [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+  ~ affiliates is strictly prohibited. Except as specified by the agreement
+  ~ under which the materials are furnished, Fiserv, Inc. and its affiliates do
+  ~ not accept any liabilities with respect to the information contained herein
+  ~ and are not responsible for any direct, indirect, special, consequential
+  ~ or exemplary damages resulting from the use of this information. No
+  ~ warranties, either express or implied, are granted or extended by this
+  ~ work or the delivery of this work.
+  -->
+
+<!--SUBJECT Developer Studio from Fiserv - Credentials Added to Workspace -->
```
Comment (anandrewfiserv) [inline email-templates/api-key-approval.template:21]: please change the default title here

File: email-templates/api-key-approval.template [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+<!--SUBJECT Developer Studio from Fiserv - Credentials Added to Workspace --> 
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Developer Studio | Credentials Added to Workspace</title>
```
Comment (anandrewfiserv) [inline email-templates/api-key-approval.template:29]: and the default title here as well

File: email-templates/api-key-approval.template (hunk 1 above)
Comment (Meet-Fiserv) [reply email-templates/api-key-approval.template]: fix it

File: email-templates/api-key-approval.template (hunk 2 above)
Comment (Meet-Fiserv) [reply email-templates/api-key-approval.template]: fix it

**Pull request merged**